dataset/src/main/kotlin/com/cosmotech/dataset/service/DatasetServiceImpl.kt

```diff
@@ -127,9 +129,15 @@ class DatasetServiceImpl(
     private val unifiedJedis: UnifiedJedis,
     private val csmRbac: CsmRbac,
     private val csmAdmin: CsmAdmin,
-    private val resourceScanner: ResourceScanner
+    private val resourceScanner: ResourceScanner,
+    @Value("\${csm.platform.twincache.useGraphModule}") private val useGraphModule: Boolean
```

Comment:
Should we document (in the openapi spec files) which endpoints are conditionally available based on that property?